Improve CMA documentation (#4264)
centreon/centreon-documentation: pp/integrations/plugin-packs/getting-started/how-to-guides/cma.md

## Introduction

The Centreon Monitoring Agent (CMA) is a piece of software installed on the host it monitors: it collects metrics and computes statuses, and sends them to Centreon.

The agent can execute native checks, or use Centreon plugins to execute non-native checks. Native checks are run directly by the agent (as opposed to non-native checks, which require local plugins to be installed on the host). Native checks have better performance and a better footprint (reduced CPU and memory usage).

> Both native and non-native checks are defined in either the **Linux Centreon Monitoring Agent** connector or the **Windows Centreon Monitoring Agent** connector. The connectors provide the templates and the agent retrieves the configuration of these checks at regular intervals after the connection has been established.

Reviewer: ```suggestion
Both native and non-native checks are defined in either the **Linux Centreon Monitoring Agent** connector or the **Windows Centreon Monitoring Agent** connector. The connectors provide the templates and the agent retrieves the configuration of these checks every times poller configuration is updated after the connection has been established.
```

<!--* TLS: the certificate is signed by a certification authority and the Common Name (CN) is verified.
* TLS insecure: the certification authority and Common Name are not verified (self-signed certificates can be used).-->

Store the certificates in the **/etc/pki** directory of the poller. Store them where you like on the host. The diagrams below describe the certificate files to be used in each case.

<Tabs groupId="sync">
<TabItem value="Agent connects to poller, TLS secure" label="Agent connects to poller, TLS secure">

![image](../../../../assets/integrations/plugin-packs/how-to-guides/cma/TLS_SEC_initiated-by-agent.png)

The poller will be configured the following way, using the **Poller/agent configuration** page, in the **OTLP receiver** section:

* Public certificate (mandatory). If you have stored the poller's certificate in the Certificate Store, you don't need to enter a file for the public key. Otherwise, you need to provide the path to the file containing the public key of the poller's opentelemetry server.
        The DNS name that the agent will use to connect to the poller must be identical to the CN of the certificate.

>         If this is not possible, you can add an IP **collector_host_name** mapping in the **C:\Windows\System32\drivers\etc\hosts** file.

Reviewer: ```suggestion
        If this is not possible, you can add an IP **collector_host_name** mapping in the **C:\Windows\System32\drivers\etc\hosts (windows)** or **/etc/hosts (linux)** file.
```

* Private key (mandatory)
* CA: rarely necessary in this case, except to manage a double handshake. The TLS protocol with certificates validates the identity of the server for the client, but the "double handshake" goes further: it adds the validation of the client's identity by the server. This is useful for enhanced security but rarely necessary on the internet.

The agent will be configured the following way on the host [(for Windows using the installer or the CLI, and for Linux using the **centagent.json** file)](#step-2-prepare-the-host).

* Encryption = yes

> * Trusted CA’s certificate file

Reviewer: ```suggestion
* Trusted CA’s certificate file (can be load in the windows certificate store)
```

> * Certificate Common Name

Reviewer: ```suggestion
* Certificate Common Name: rarely necessary
```

   CA and CA Common Name (CN) are optional.

   * CA : In the case of a pubic certificate, the standard OS certification chain is sufficient, this parameter is not required.
   In the case of a self-signed certificate, the CA can be added to the OS in its certification chains, making this parameter unnecessary. If you do not add the CA to the OS, fill in the CA field.

   * CN : The poller uses a domain name or IP to connect to the agent. If the certificate used on the agent matches this domain/IP, then leave the field blank. If it doesn't match, fill in the field.

   Please note that the CN field in the certificate must match the name that will be used by the poller to connect to the host. For example, if you have entered **myhostname** in the CN, the poller must be able to connect to the host **myhostname** without using the IP address (a solution if **myhostname** is not in the DNS: add the IP myhostname mapping in the **/etc/hosts** file).

The agent will be configured the following way on the host [(for Windows using the installer or the CLI, and for Linux using the **centagent.json** file)](#step-2-prepare-the-host).

* Encryption = yes

> * Certificate file

Author: ```suggestion
* Public certificate file
```
